modules/@apostrophecms/rich-text-widget/ui/apos/tiptap-extensions/Heading.js

```diff
@@ -22,6 +22,16 @@ export default (options) => {
           rendered: false
         }
       };
+    },
+    addKeyboardShortcuts() {
+      const marks = Object.keys(this.editor.schema.marks);
+      return this.options.levels.reduce((items, level) => ({
+        ...items,
+        ...{
+          [`Mod-Alt-${level}`]: () => this.editor.commands.toggleHeading({ level }),
```

Comment:
@boutell this bit is boilerplate from tiptap's heading extension, maybe there is a way to avoid the duplication?